cmd/user-agent/src/main.js

```diff
@@ -32,17 +35,44 @@ async function loadAriesOnce() {
     return Vue.prototype.$aries
 }
 
-function ariesStartupOpts() {
-    // TODO start up option should be fetched from environment
+
+let defaultAriesStartupOpts = {
+    assetsPath: '/aries-framework-go/assets',
+    'outbound-transport': ['ws', 'http'],
+    'transport-return-route': 'all',
+    'http-resolver-url': [],
+    'agent-default-label': 'demo-user-agent',
+    'auto-accept': true,
+    'log-level': 'debug',
+    'db-namespace': 'agent'
+}
+
+async function ariesStartupOpts() {
+    let startupOpts = {}
+
+    if (process.env.NODE_ENV === "production") {
+        const axios = require('axios').default;
+
+        // call service to get the opts
+        await axios.get(window.location.origin + '/aries/jsopts')
```

Comment:
TODO: probably could improve the path...

Reply:
sure. any suggestions ?